Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "zang"
+version = "0.1.0"
+edition = "2021"
```

Comment:
I think we have 2023 edition 🚀

Reply:
the Only newer edition is 2024 which I think is unstable at the moment: 
[Reference](https://doc.rust-lang.org/stable/cargo/reference/unstable.html#edition-2024)